ref/addons.md

```diff
@@ -25,6 +25,7 @@ The following table lists the valid keys in an addon manifest:
 | `description`         | string       | Description of the addon.                                                                                                                                                                                                     |                                       |
 | `provides`            | object       | Additional capabilities that this addon provides to the container. See individual items for details.                                                                                                                          |                                       |
 | `provides.abi-support`| string array | Tells AMS that this addon adds support for the given architecture even if the application doesn't support it natively. Use this when your addon brings instruction translation or provides libraries for other architectures. | `arm64-v8a`, `armeabi-v7a`, `armeabi` |
+| `hooks.timeout`| string | Execution timeout for each hook that is included in an addon. By default, the timeout is set to 5 minutes. It can be extended up to 15 minutes. Configure this option if a hook takes longer than 5 minutes to finish. | `10m` |
```

Comment:
Sorry, one more nitpick:
```suggestion
| `hooks.timeout`| string | Execution timeout for each hook that is included in an addon. By default, the timeout is set to 5 minutes. It can be extended to up to 15 minutes. Configure this option if a hook takes longer than 5 minutes to finish. | `10m` |
```
You need the two "to" in there - "up to 15 minutes" because it can be 15 or less, and "extended to" because if you had a fixed number, you would also say "extended to 15 minutes" and not "extended 15 minutes".